src/NuGet.Core/NuGet.PackageManagement/Strings.resx

```diff
@@ -487,4 +487,8 @@
   <data name="XdtError" xml:space="preserve">
     <value>An error occurred while applying transformation to '{0}' in project '{1}'</value>
   </data>
+  <data name="SignatureVerificationMultiple" xml:space="preserve">
+    <value>Multiple signature verification errors {0}</value>
```

Comment:
You need a verb.  How about the following?  (Note the space at the end of the line has been replaced with a colon.)

    Signed package validation failed with multiple errors:{0}